journal.py

```diff
@@ -57,11 +59,20 @@ def all(cls, session=None):
         return session.query(cls).order_by(cls.created.desc()).all()
 
     @classmethod
-    def get_entry(cls, entry_id, session=None):
+    def last(cls, session=None):
         if session is None:
             session = DBSession
-        entry = session.query(cls).filter(Entry.id == entry_id).first()
-        return entry.title, entry.text, entry.created.strftime('%b. %d, %Y')
+        return session.query(cls).order_by(cls.created.desc()).first()
+
+    @classmethod
+    def get(cls, entry_id=None, title=None, session=None):
+        if session is None:
+            session = DBSession
+        if title is None:
+            entry = session.query(cls).filter(Entry.id == entry_id).first()
+        else:
+            entry = session.query(cls).filter(Entry.title == title).first()
+        return entry
```

Comment:
Why give yourself all these options?  You should be able to pass the ID of the entry around and just use that.  It's a cleaner match than the title anyway, since it's the primary key.  And it's faster, since the title is not indexed.